docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,159 @@
 Tutorial 5 - Taking it mobile: Android
 ======================================
 
-To deploy our applicaton to Android...
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. There is one additional
+pre-requisite: Java 8.
+
+Setting up Java 8
+=================
+
+To use BeeWare's Android tooling, you must install a Java 8 compiler and provide
+the path to it in the ``JAVA_HOME`` environment variable. This is because
+BeeWare's Android tooling depends on the Android SDK, and the Android SDK
+depends on the Java runtime and compiler being available. Some tools within
+the Android SDK specifically require version 8 of Java, so we stick to that
+in this documentation.
+
+Download & install Java 8
+-------------------------
+
+Visit the `AdoptOpenJDK website <https://adoptopenjdk.net/>`_ and
+download OpenJDK 8.
+
+On macOS, the default format is a *.pkg* installer. Download and execute the
+installer, accepting the defaults. In your terminal, run:
+``export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"``
```

Comment:
Need to flag that this needs to be done every time you start a terminal session; or, be added to your .bash_profile.